azure/packages/azure-client/package.json

```diff
@@ -45,31 +45,31 @@
 	"dependencies": {
 		"@fluidframework/common-definitions": "^0.20.1",
 		"@fluidframework/common-utils": "^1.1.1",
-		"@fluidframework/container-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
-		"@fluidframework/container-loader": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
-		"@fluidframework/core-interfaces": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
-		"@fluidframework/driver-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
-		"@fluidframework/driver-utils": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
-		"@fluidframework/fluid-static": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
-		"@fluidframework/map": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/container-definitions": ">=2.0.0-internal.4.2.0 <2.0.0-internal.5.0.0",
```

Comment:
Shouldn't this be < internal.4.3.0?

Reply:
I don't think these ranges really matter since they get replaced when publishing now. Soon they should just be `workspace:*`